src/julia_gc.c

```diff
@@ -826,16 +832,26 @@ void InitBags(UInt initial_size, Bag * stack_bottom, UInt stack_align)
     jl_gc_set_cb_post_gc(PostGCHook, 1);
     // jl_gc_enable(0); /// DEBUGGING
 
+    // Get GapObj abstract julia type
+    jl_module_t * gaptypes_module =
+        (jl_module_t *)jl_get_global(jl_main_module, jl_symbol("GAPTypes"));
+    jl_datatype_t * gapobj_type =
+        (jl_datatype_t *)jl_get_global(gaptypes_module, jl_symbol("GapObj"));
+
     Module = jl_new_module(jl_symbol("ForeignGAP"));
```

Comment:
We have `GAPTypes.jl`, `GAP.jl`, `ForeignGAP` vs. `GapObj` ... Should we rename the latter to `GAPObj` to make things more consistent?

Reply:
I do not really care tbh. `GAPObj` looks more consistent, but less Julia . . .

Reply:
We should either decide between `GapObj` and `GAPObj` now; or else, be willing to potentially carry both of them around for backwards compatibility (which is fine, IMHO, just mentioning it). In either case, I am not worried :-)

Reply:
I am totally happy with both spellings, too.